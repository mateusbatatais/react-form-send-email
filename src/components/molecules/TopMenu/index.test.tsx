import React from "react";
import { render } from "@testing-library/react";
import TopHeader from "./index";

describe("Testa top header", () => {
  it("snapshot", () => {
    const { container } = render(<TopHeader />);
    expect(container).toMatchSnapshot();
  });
});
