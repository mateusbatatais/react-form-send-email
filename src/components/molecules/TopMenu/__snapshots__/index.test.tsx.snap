// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testa top header snapshot 1`] = `
<div>
  <header
    class="bgBlack text-light-custom"
  >
    <div
      class="container"
    >
      <div
        class="menuNav menuNav text justify-content-end py-4 nav"
      >
        <div
          class="nav-item"
        >
          <a
            class="nav-link"
            data-rr-ui-event-key="/"
            href="/"
          >
            Home
          </a>
        </div>
        <div
          class="nav-item"
        >
          <a
            class="nav-link"
            data-rr-ui-event-key="/listagem"
            href="/listagem"
          >
            Itens cadastrados
          </a>
        </div>
        <div
          class="nav-item"
        >
          <a
            class="nav-link"
            data-rr-ui-event-key="/"
            href="/"
          >
            Como funciona
          </a>
        </div>
        <div
          class="nav-item"
        >
          <a
            class="nav-link"
            data-rr-ui-event-key="/"
            href="/"
          >
            Privacidade
          </a>
        </div>
        <div
          class="nav-item"
        >
          <a
            class="nav-link"
            data-rr-ui-event-key="/"
            href="/"
          >
            Ajuda
          </a>
        </div>
      </div>
    </div>
  </header>
</div>
`;
